src/dug/core/async_search.py

```diff
@@ -690,3 +418,134 @@ async def search_kg(self, unique_id, query, offset=0, size=None,
         )
         search_results.update({'total_items': total_items['count']})
         return search_results
+
+    def get_query(self, concept, fuzziness, prefix_length, query):
```

Comment:
It would be great to have some documentation on what this function does -- in particular, how it differs from `_build_concepts_query()`, which looks very similar.

Reply:
i will rename and add a comment. It will add some clarity. Maybe someone else could provide more insight in differences, i don't have this info yet